api/controllers/v0.3/endpoints.js

```diff
@@ -378,14 +378,11 @@ async function createPatron(req, res) {
           ...card.details(),
         };
       } catch (error) {
-        console.log(
-          "controller createPatron error calling card.createIlsPatron",
-          error
-        );
         logger.error(
           "controller createPatron error calling card.createIlsPatron",
           error
         );
+        logger.info("errored patron info", card);
```

Comment:
Can you be more specific about what's being logged so it doesn't include all the card's information but only the relevant ones?

Reply:
I do want to see the whole request that's being sent. I guess we could start with just the pin, but I have a feeling that it could be an issue with the request in general, and not just the pin.